content/gocourse-2019-05-melbourne.slide

```diff
@@ -813,49 +816,106 @@ Embedding types introduces the possiblity for naming conflicts. Go follows some
 .caption J. D. Frazer "Illiad", User Friendly - [[https://en.wikipedia.org/wiki/User_Friendly#Crud_Puppy][Crud Puppy]]
 
 
-* Errors (1/3)
+* Errors in Go
 
-"Errors are values."  - [[https://go-proverbs.github.io/][Rob Pike]]
-
-The `error` type is a built-in interface:
+- `error` a builtin interface ([[https://golang.org/src/builtin/builtin.go?s=11246:11286#L251][source: error]])
```

Comment:
s/ a/ is (as @anzboi mentioned on slack).

Reply:
love the linke to the source!

Reply:
Thanks I've updated that now. Pushed it to git but haven't yet published.